servers/Azure.Mcp.Server/README.md

```diff
@@ -598,9 +598,27 @@ The software may collect information about you and your use of the software and
 <!-- remove-section: start vsix remove_telemetry_config_section -->
 ### Telemetry Configuration
 
-Telemetry collection is on by default.
+Telemetry collection is on by default. The server supports two telemetry streams:
 
-To opt out, set the environment variable `AZURE_MCP_COLLECT_TELEMETRY` to `false` in your environment.
+1. **User-provided telemetry**: If you configure your own Application Insights connection string via the `APPLICATIONINSIGHTS_CONNECTION_STRING` environment variable, telemetry will be sent to your Application Insights resource.
+
+2. **Microsoft telemetry**: By default, telemetry is also sent to Microsoft to help improve the product. This can be disabled separately from user-provided telemetry.
```

Comment:
Maybe link to a privacy statement?

Reply:
Will update this section in my next PR which has more doc changes.